test/jest_integration/saml_auth.test.ts

```diff
@@ -244,7 +244,7 @@ describe('start OpenSearch Dashboards server', () => {
     await driver.wait(until.elementsLocated(By.xpath(pageTitleXPath)), 10000);
 
     const cookie = await driver.manage().getCookies();
-    expect(cookie.length).toEqual(2);
+    expect(cookie.length).toEqual(3);
```

Comment:
Is modifying this to 3 robust enough here? I believe the assertion here looks for 2 cookies corresponding to the 2 different origins. OSD and the external SAML IdP - in this case the node-based SAML IdP used for functional tests. Would it be possible to check for at least 2 cookies here and verify its from 2 different origins?